jenkins/shared-library/vars/publish.groovy

```diff
@@ -195,3 +196,15 @@ boolean isGitHubRepositoryPublic(String orgName, String repoName) {
 		return false
 	}
 }
+
+void awsPublisher(Map config, String phase) {
+	if (config.publisher != 'aws' || !isRelease(phase)) {
+		return
+	}
+
+	withCredentials([usernamePassword(credentialsId: config.awsCredentialId,
+			usernameVariable: 'AWS_ACCESS_KEY_ID',
+			passwordVariable: 'AWS_SECRET_ACCESS_KEY')]) {
```

Comment:
notice indentation in file is inconsistent - in some cases these are indented twice, others once

Reply:
yea, its my editor doing auto-format. 😬 
I need to switch back to vi 😄